k8s_authorization/README.md

```diff
@@ -0,0 +1,63 @@
+# Kubernetes authorization webhook using OPA
+
+Runnable Kubernetes
+[authorization webhook](https://kubernetes.io/docs/reference/access-authn-authz/webhook/)
+example using OPA for authorization policy decisions.
+
+## Running
+
+Just run:
```

Comment:
[nit] It's only "just run" if you've got kind and kubectl installed, right? I guess the latter can be taken for granted and the former is mentioned below, so we're good. 😄

Reply:
Ah, thanks.. I'll add a "prerequisites" section before that.